setup.py

```diff
@@ -48,17 +44,19 @@ def read(*names, **kwargs):
     keywords=['geocoding', 'gis'],
     install_requires=['requests==2.23.*', 'docopt==0.6.*', 'gitpython==3.1.*'],
```

Comment:
**chore**: the geocoding only really needs docopt and requests to run. gitpython is not required and should be removed. We should import it only in the branch that requires it so the script can run without issue.

Reply:
`geocoding.py` doesn't use docopt, does it?